poleno/utils/templatetags/poleno/amend.py

```diff
@@ -244,3 +245,47 @@ def action(fragment):
 
     context[u'_amend'].append(action)
     return u''
+
+@register.simple_tag(takes_context=True)
+def set_attributes(context, path, **kwargs):
+    u"""
+    Select elements specified by XPath and add, remove or edit their attributes.
+
+    Example:
+        {% amend %}
+          <ul>
+            <li>xxx</li>
+            <li ccc="value" ddd>yyy</li>
+            <li eee="value">zzz</li>
+          </ul>
+          {% set_attributes path=".//li[1]" aaa="value" bbb=True %}
+          {% set_attributes path=".//li[2]" ccc=None ddd=False %}
+          {% set_attributes path=".//li[3]" eee="new_value" %}
```

Comment:
Tie examples mas tiez hodne chaoticke. Sprav to tak, aby pri kazdom elemente si spravil len jednu operaciu. A postupne prejdi vsetkymi operaciami:

```
<ul>
  <li aaa="bbb">ccc</li>
  <li aaa="bbb">ccc</li>
  <li aaa="bbb">ccc</li>
  <li aaa="bbb">ccc</li>
</ul>
{% set_attributes path=".//li[1]" aaa=None xxx=None %}
{% set_attributes path=".//li[2]" aaa=True xxx=True %}
{% set_attributes path=".//li[3]" aaa=False xxx=False %}
{% set_attributes path=".//li[4]" aaa="foo" xxx="bar" %}
```

Reply:
:ok: